improver_tests/cube_combiner/test_CubeMultiplier.py

```diff
@@ -144,6 +160,37 @@ def test_exception_for_single_entry_cubelist(self):
         with self.assertRaisesRegex(ValueError, msg):
             plugin.process(cubelist, "new_cube_name")
 
+    def test_update_cell_methods(self):
+        """Test that plugin updates cell methods where required when a new
+        diagnostic name is provided."""
+        cubelist = iris.cube.CubeList([self.cube, self.multiplier])
+
+        new_cube_name = "new_cube_name"
+        expected = CellMethod("sum", coords="time", comments=f"of {new_cube_name}")
+
+        result = CubeMultiplier()(cubelist, new_cube_name, broadcast_to_threshold=True)
+        self.assertEqual(result.cell_methods[0], expected)
+
+    def test_unmodified_cell_methods(self):
+        """Test that plugin leaves cell methods that are diagnostic name
+        agnostic unchanged."""
+
+        cell_methods = list(self.cube.cell_methods)
+        additional_cell_method = CellMethod("sum", coords="longitude")
```

Comment:
If I change this as below, the test fails.
```suggestion
        additional_cell_method = CellMethod("sum", coords="longitude", comments="Does not match cube name")
```

Reply:
Issue fixed and test extended as written above.